configure.ac

```diff
@@ -662,7 +662,7 @@ fi
 # Check for stream support via WinPR
 if test "x${have_freerdp}" = "xyes"
 then
-    AC_CHECK_HEADER(winpr/stream.h,,
+    AC_CHECK_HEADER(winpr/stream.h,[RDP_LIBS="$RDP_LIBS -lwinpr-utils"],
```

Comment:
This is unfortunately probably wrong.

This section is specific to one aspect of that WinPR library, so adding the library here is odd. It would make more sense to add the WinPR library to `RDP_LIBS` at the point where the final yes/no decision is made regarding whether WinPR-related parts of the build are enabled, a little bit later:

```
if test "x${have_freerdp}" = "xyes" -a "x${have_winpr}" = "xyes"
then
    AC_DEFINE([ENABLE_WINPR],,
              [Whether library support for WinPR types was found])
fi
```

I have not looked into the original issue further, but I'm also unsure whether "libwinpr-utils" is the correct library to add.